python_ta/typecheck/base.py

```diff
@@ -127,6 +156,14 @@ def op_to_dunder_unary(op):
         return op
 
 
+def _correct_article(noun : str) -> None:
```

Comment:
The return type is a `str`, not None. (Didn't PyCharm give you a warning about this?)